feat(docs): added skip link component documentation (#1210)
dequelabs/cauldron: docs/pages/components/SkipLink.mdx

---
title: SkipLink

> description: The component is a user interface element designed to enhance website accessibility and usability. By clicking the skip link, users can efficiently navigate to the core content, improving their browsing experience and ensuring compliance with accessibility standards.

Reviewer: Just a small suggestion to provide a little bit better context for what this component is.

```suggestion
description: A navigation component to provide quick access to main content or other areas of the page for screen reader and keyboard users.
```

source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/SkipLink/index.tsx
---

import { SkipLink } from '@deque/cauldron-react'

```jsx
import { SkipLink } from '@deque/cauldron-react'
```

> The skip link is at the very top of the page, but it’s “hidden”. The link will be “activated” by navigating through the page with the Tab button right after the page is loaded.

Reviewer: ```suggestion
The skip link is visually located at the very top of the page, but it’s hidden. The link will be activated by navigating through the page with the Tab key based on its document location.
```

Nit, I don't think the quotes are necessary. Additionally, the skip link will be located in the the tab order based on its document order.

## Example

```jsx example
<SkipLink target="#main-content" skipText="Skip to" targetText="Main page content" />
```

## Props

> ### Step

Reviewer: ```suggestion
```

Reviewer: Noticed this extra heading here.

> <SkipLink target="#main-content" aria-label={"Skip"} />

Reviewer: `aria-label` will override the internal labels, can we use the `skipText` and/or `targetText` props instead?

<ComponentProps
  props={[
    {
      name: 'target',
      type: 'string',
      required: true,

>       description: 'Id of the element. It uses to navigate to the element.'

Reviewer 2: `Id of the element to which a skip link navigates`?

    },
    {
      name: 'skipText',
      type: 'string',

>       description: 'Text, which uses to indicate that the block is used to skip to',

Reviewer 2: Is it a `Text label for the skip link`?

      default: 'Skip to'
    },
    {
      name: 'targetText',
      type: 'string',

>       description: 'Text to indicate to which content the link navigate to',

Reviewer 2: text label for the target?
